runc.go

```diff
@@ -287,7 +287,7 @@ func (r *Runc) Exec(context context.Context, id string, spec specs.Process, opts
 // Run runs the create, start, delete lifecycle of the container
 // and returns its exit status after it has exited
 func (r *Runc) Run(context context.Context, id, bundle string, opts *CreateOpts) (int, error) {
-	if opts.Started != nil {
+	if opts != nil && opts.Started != nil {
```

Comment:
L309 has the same issue

Reply:
Probably we should just do `if opts == nil { opts = &CreateOpts} ` and remove all `if opts != nil` checks

Reply:
`Exec()` also has the same issue